rye/src/cli/init.rs

```diff
@@ -34,6 +36,8 @@ pub struct Args {
     /// Which build system should be used?
     #[arg(long, default_value = "hatchling")]
     build_system: BuildSystem,
+    #[arg(long, default_value = "MIT")]
+    license: String,
```

Comment:
I think the license choice should be opt-in. Let's make this `Option<String>` instead and only create the license if it's passed explicitly.

Reply:
had modify to using `Option<String>` for license.